src/mocks/data/user-profile.json

```diff
@@ -0,0 +1,44 @@
+{
+  "bio": "Developing Jest Preview.",
+  "blog": "https://hung.dev",
+  "company": "Looking for job",
```

Comment:
It's cool, we have a snapshot of your profile riight in _Best of JS_ 😄 

I'm pretty sure there you will belong to a great company soon!